BICI_Psoc_workspace/Fingertip_V3_acc.cydsn/main.h

```diff
@@ -13,14 +13,27 @@
 #include "project.h"
 #include <stdbool.h>
 
-#define TAXEL_COUNT         (67)
+
+#define TAXEL_COUNT         (70)

```

Comment:
Would be nice to use a different variable to know what's happening.

Reply:
I added  ACC_DATA_SIZE